sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -122,10 +129,20 @@ export class IdentityClient extends ServiceClient {
         // It's likely that the refresh token has expired, so
         // return null so that the credential implementation will
         // initiate the authentication flow again.
+        span.setStatus({
+          code: CanonicalCode.UNAUTHENTICATED,
+          message: err.message
+        })
```

Comment:
Nit: missing semi-colon.

Reply:
Good eyes! Definitely need to get lint working...